Source/WebCore/mathml/MathMLSelectElement.cpp

```diff
@@ -164,7 +165,7 @@ Element* MathMLSelectElement::getSelectedSemanticsChild()
     if (!child)
         return nullptr;
 
-    if (!is<MathMLElement>(*child) || !downcast<MathMLElement>(*child).isPresentationMathML()) {
+    if (auto* childElement = dynamicDowncast<MathMLElement>(*child); childElement && childElement->isPresentationMathML()) {
```

Comment:
Still not quite right; it should be `or`, not `and` to keep the old behavior like:
`if (auto* childElement = dynamicDowncast<MathMLElement>(*child); !childElement || childElement->isPresentationMathML())`

Reply:
I may have typed it wrong in my earlier review comment, if so, I apologize.

But @szewai still typed it wrong again I think :p, both conditions should use `!`:
```
if (auto* childElement = dynamicDowncast<MathMLElement>(*child); !childElement || !childElement->isPresentationMathML()) {
```

Reply:
ah nice catch; not sure the new change is better then since it's so easy to mistype : P!